src/chi-vue/src/constants/classes.ts

```diff
@@ -141,6 +141,7 @@ export const GENERIC_SIZE_CLASSES = {
 
 //#region Icon
 export const ICON_CLASS = 'chi-icon';
+export const INFO_ICON_CLASS = 'icon-circle-info-outline';
```

Comment:
We don't put icon classes in `classes.ts` as there are too many icons.
In a recent [PR](https://github.com/CenturyLink/Chi/pull/241) Soujan created a file `icons.ts` to put the icon class names.
We agreed that it's the best thing to do, otherwise `classes.ts` will get too long and unreadable.
Please, review peer PRs and adopt new patterns.

Reply:
oh sorry i didn't notice that file, thanks